packages/next/client/link.tsx

```diff
@@ -241,7 +240,7 @@ class Link extends Component<LinkProps> {
         }
         this.prefetch()
       },
-      onClick: (e: React.MouseEvent) => {
+      onClick: (e: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
This might cause type errors for existing projects that use `<Link>` with non-anchor elements, right?

Reply:
@Timer I'm not 100% sure what the effect would be, but to play safe I replaced it with a type cast.